<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MnsClass</class>
 <widget class="QWidget" name="MnsClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>908</width>
    <height>537</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>27</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>32</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>17</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QWidget" name="widget" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>57</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: blue;</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>27</number>
     </property>
     <property name="leftMargin">
      <number>56</number>
     </property>
     <item>
      <widget class="QPushButton" name="button_alias">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>49</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>150</width>
         <height>49</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: #b6a27e;
color: #f2f2f2</string>
       </property>
       <property name="text">
        <string>START ALIAS</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="button_all">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>142</width>
         <height>49</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>142</width>
         <height>49</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: #b6a27e;
color: #f2f2f2</string>
       </property>
       <property name="text">
        <string>START ALL</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="button_missing">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>191</width>
         <height>49</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>191</width>
         <height>49</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: #b6a27e;
color: #f2f2f2</string>
       </property>
       <property name="text">
        <string>START MISSING</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="button_update">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>191</width>
         <height>49</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>191</width>
         <height>49</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: #b6a27e;
color: #f2f2f2</string>
       </property>
       <property name="text">
        <string>UPDATE STATUS</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="leftMargin">
      <number>56</number>
     </property>
     <item>
      <widget class="QLabel" name="label_notify1">
       <property name="styleSheet">
        <string notr="true">background-color: transparent;
color: #f2f2f2</string>
       </property>
       <property name="text">
        <string>Status will be updated automatically in</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_notify2">
       <property name="styleSheet">
        <string notr="true">background-color: transparent;
color: #b99d7c;
padding-left: 6px;
padding-right: 5px;</string>
       </property>
       <property name="text">
        <string>48</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_notify3">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>18</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>18</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: transparent;
color: #f2f2f2</string>
       </property>
       <property name="text">
        <string>SECS</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
